internal/groups/storage_test.go

```diff
@@ -65,7 +66,7 @@ var _ = Describe("Storage", func() {
 			}
 
 			return r
-		}).Should(And(ContainElement("a"), ContainElement("b")))
+		}).Should(And(ContainElement("a"), ContainElement("b"), ContainElement("c")))
```

Comment:
Should there be more elemenats than "a", "b", and "c". If not can we switch this to `ConsistOf`

Reply:
`ConsistOf` does not take into account that each source id will be egressed multiple times.

This is from where I switched to `ConsistOf`:
```
Timed out after 1.000s.
  Expected
      <[]string | len:4, cap:4>: ["b", "c", "b", "a"]
  to consist of
      <[]interface {} | len:3, cap:3>: ["a", "b", "c"]
```